projects/legacy/components/input-tag/input-tag.style.less

```diff
@@ -285,10 +285,7 @@
 }
 
 @media @tui-mouse {
-    :host:not(._readonly):not(._disabled):hover
-        .t-textfield-icon
-        ::ng-deep
-        [tuiAppearance][data-appearance='icon']::after {
+    :host:not(._readonly):not(._disabled):hover .t-textfield-icon ::ng-deep [tuiAppearance][data-appearance='icon']::after {
```

Comment:
Weird that this got affected 🤔 You should rollback those style changes. Maybe you need to run `npm i` to update some lint config packages locally?